Blog/views.py

```diff
@@ -126,7 +126,10 @@ def user_detail_view(request, pk):
     user = User.objects.get(pk=pk)
     posts = Post.objects.filter(user=user)
     comments = Comment.objects.filter(user=user)
-    liked_posts = Post.objects.all().first().votes.all(user.id, action=UP)
+    if Post.objects.all():
```

Comment:
`django-vote` adds a special model manager ([see docs](https://docs.djangoproject.com/en/2.0/topics/db/managers/)) called `votes` on all votable models which could be used to perform the database query: `Post.votes.all(user.pk, action=UP)`

This should return an empty queryset if there are no blog posts (but I did not verify it).